install_config/configuring_gce.adoc

```diff
@@ -75,4 +134,60 @@ container. Therefore, *_node-config.yaml_* should be in *_/etc/origin/node_*
 instead of *_/etc/_*.
 ====
 
-Then, start or restart the {product-title} services on the master and all nodes.
+. Start or restart the {product-title} services all nodes.
++
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-node
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-node
+----
+endif::[]
+
+
+[[configuring-gce-multizone]]
+
+== Configuring Multizone Support in a GCE Deployment
+
+// From https://bugzilla.redhat.com/show_bug.cgi?id=1390160
+
+If manually congifuring GCE, multizone support is not configured by default. 
+
+[NOTE]
+====
+The advanced installation configures multizone support by default. 
+====
+
+If you want multizone support:
+
+. Edit or create a *_/etc/origin/cloudprovider/gce.conf_* file on all of your {product-title} hosts, both masters and nodes. 
+. Add the following contents:
++
+====
```

Comment:
You can remove the ==== around the code block and just use the ---- 

(I need to update the doc guidelines to discuss this) :pencil2: